astroquery/casda/core.py

```diff
@@ -0,0 +1,137 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+# put all imports organized as shown below
+# 1. standard library imports
+from six import BytesIO
+
+# 2. third party imports
+import astropy.units as u
+import astropy.coordinates as coord
+from astropy.table import Table
+
+# 3. local imports - use relative imports
+# commonly required local imports shown below as example
+# all Query classes should inherit from BaseQuery.
+from ..query import BaseQuery
+# has common functions required by most modules
+from ..utils import commons
+# prepend_docstr is a way to copy docstrings between methods
+from ..utils import prepend_docstr_nosections
+# async_to_sync generates the relevant query tools from _async methods
+from ..utils import async_to_sync
+# import configurable items declared in __init__.py
+from . import conf
+
+# export all the public classes and methods
+__all__ = ['Casda', 'CasdaClass']
+
+
+@async_to_sync
+class CasdaClass(BaseQuery):
+
+    """
+    Class for accessing ASKAP data through the CSIRO ASKAP Science Data Archive (CASDA). Typical usage:
+
+    result = Casda.query_region('22h15m38.2s -45d50m30.5s', radius=0.5 * u.deg)
+    """
+    # use the Configuration Items imported from __init__.py to set the URL,
+    # TIMEOUT, etc.
+    URL = conf.server
+    TIMEOUT = conf.timeout
+
+    # all query methods are implemented with an "async" method that handles
+    # making the actual HTTP request and returns the raw HTTP response, which
+    # should be parsed by a separate _parse_result method.   The query_object
+    # method is created by async_to_sync automatically.
+    def query_region_async(self, coordinates, radius=None, height=None, width=None,
+                           get_query_payload=False, cache=True,
+                           **kwargs):
```

Comment:
kwargs is not used later on, please remove and add `verbose` directly if you plan to use it, or remove it, too from the parameters list from below, if it's not used.